CHANGELOG.md

```diff
@@ -4,15 +4,23 @@
 - **Breaking:** Remove `linkId` prop from `<Breadcrumb.Item>`, for consistency with other wrapped link components ([#2011])
 - **Breaking:** Remove deprecated functionality from v0.29.x ([#2017])
 - **Breaking:** Remove redundant `caption` prop from `<Carousel.Item>`, as this functionality is available from `<Carousel.Caption>` ([#2018])
+- **Breaking:** Remove pointless `navItem` and `navDropdown` props from `<Button>` ([#2026])
```

Comment:
slightly related... do we cover `navbar-btn` anywhere? that's definitely a thing I wonder if this was a bad attempt at that?